examples/hotel_ads/add_hotel_ad.py

```diff
@@ -267,8 +270,12 @@ def add_hotel_campaign(
         "-b",
         "--bid_ceiling_micro_amount",
         type=int,
-        required=True,
-        help=("The bid ceiling micro amount for " "the hotel campaign."),
+        required=False,
+        default=_CPC_BID_CEILING_MICRO_AMOUNT,
+        help=(
+            "The bid ceiling micro amount for the hotel campaign. "
+            "The default value is 20000000"
```

Comment:
should this use the `_CPC_BID_CEILING_MICRO_AMOUNT` variable instead of hardcoding 20000000?

Reply:
Good idea!